ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -61,6 +61,7 @@ class CommandBarButton: UIButton {
         setTitle(iconImage != nil ? nil : item.title, for: .normal)
         titleLabel?.isEnabled = isEnabled
         titleLabel?.font = item.titleFont
+        accessibilityLabel = item.accessibilityLabel
```

Comment:
Quite often a display string will not suffice as an accessibility label, because they tend to be shorter and provide less contextual information. The existing APIs set the accessibility label from a related property on the CommandBarItem which is passed in the init function. For consistency we would need to update that code also, but I think it is probably better to have the consumer explicitly set the value they want used.
In the case of how we currently consume it, the strings come from the same source, but I am working to have separate accessibility strings in many cases.
That said, if you ultimately prefer to have the fallback logic I can add that.

Reply:
I think we should have the fall back logic because that is the behavior of existing UIButton or UITabbar. most people won't think about it and just expect it will work by default.

Reply:
No problem, I will make that extra change shortly, thanks!